bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -1401,13 +1401,33 @@ impl Client {
     }
 
     /// Get the media previews timeline display policy
+    /// currently stored in the cache.
+    pub async fn get_stored_media_preview_display_policy(
```

Comment:
The `stored` is a bit rendundant here. We have a kinda soft policy that we call functions that get things from the store `get_foo()` while we call functions that get things from the server `fetch_foo()`.

```suggestion
    pub async fn get_media_preview_display_policy(
```